src/Service/Kraken/KrakenWithdrawService.php

```diff
@@ -12,6 +12,9 @@
 
 class KrakenWithdrawService implements WithdrawServiceInterface
 {
+
+  public static $ASSETS = array("ETH"=>"XETH", "ADA"=>"ADA", "XBT"=>"XXBT");
```

Comment:
Dit zouden we ergens in een config kunnen plaatsen en vervolgens deze variable vullen in de _construct(). Deze tokens staat nu namelijk op 3 plekken benoemd:
1. in dit bestand
2. In .env 
3. In config/services.yml

Reply:
Ik heb het verplaatst naar de services.yaml